src/common/host_device_vector.cc

```diff
@@ -93,16 +98,16 @@ common::Span<const T> HostDeviceVector<T>::ConstDeviceSpan(int device) const {
 }
 
 template <typename T>
-std::vector<T>& HostDeviceVector<T>::HostVector() { return impl_->data_h_; }
+std::vector<T>& HostDeviceVector<T>::HostVector() { return impl_->Vec(); }
 
 template <typename T>
 const std::vector<T>& HostDeviceVector<T>::ConstHostVector() const {
-  return impl_->data_h_;
+  return impl_->Vec();
```

Comment:
I see lots of replacement of member access with getters. Does clang-tidy always prefer getters/setters to direct member access?

Reply:
No.  Private member is required to add a "_" suffix while public member should not contain such suffix.  So sometimes I change public member into private in order to preserve the suffix.